splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkIntegration.java

```diff
@@ -86,6 +88,9 @@ private void configureErrorHandler() throws Exception {
         onException(IOException.class)
             .to(direct("error"))
             .handled(true);
+        onException(HttpOperationFailedException.class)
+            .to(direct("error"))
```

Comment:
Can you create a new handler for these and get it to log the status code, body, and headers? More info about what comes with HttpOperationFailedException is in https://www.javadoc.io/doc/org.apache.camel/camel-http4/2.8.0/org/apache/camel/component/http4/HttpOperationFailedException.html.

specifically I would try to grab:

getResponseBody()
getResponseHeaders()
getStatusCode()
getStatusText()
getUri()

and log all of them